lib/rubies/game.rb

```diff
@@ -107,6 +107,16 @@ def itsright
       puts "Correct!".colorize(:green)
     end
 
+    def cheater
+      @num_wrong += 1
+      puts "Please provide an answer "\
+            "that operates on the given values. ".colorize(:light_red)
+      puts
+      puts "Try not to directly reference the value directly, ".colorize(:light_red)
```

Comment:
Line is too long. [84/80]<br>Do not write to stdout. Use Rails' logger if you want to log.